### ISSUE TITLE
scipy.fftpack was renamed to scipy.fft

### ISSUE THREAD

**Issue by fxhuhn** [JohannesBuchner/imagehash#108]
Hi there,

the  submodule scipy.fftpack was renamed to scipy.fft, I guess there were no other major changes. 
Can you update this?

Regards Markus

**Comment by JohannesBuchner** [comment]
Good idea. Can you submit a brief pull request? Travis should check whether the tests succeed.

**Comment by rosanzheng** [comment]
Is there a way to work around this currently? I tried downgrading my installed scipy package, but breaks my environment with tensorflow :/

**Comment by JohannesBuchner** [comment]
help with https://github.com/JohannesBuchner/imagehash/pull/201 and test it
